system_tests/app_engine/test_app_engine.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+from google.auth import _cloud_sdk
+import pytest
+
+
+HERE = os.path.dirname(__file__)
+TEST_APP_DIR = os.path.join(HERE, 'app')
+TEST_APP_SERVICE = 'google-auth-system-tests'
+
+
+def vendor_app_dependencies():
+    """Vendors in the test application's third-party dependencies."""
+    subprocess.check_call(
+        ['pip', 'install', '-t', 'lib', '-r', 'requirements.txt'])
```

Comment:
> Holy moly this is happening.

![tumblr_inline_nzb6a3hkjg1rp5ay0_500](https://cloud.githubusercontent.com/assets/250995/19782096/327e8478-9c41-11e6-96c6-57964f48423f.gif)

> Are there long forms of -t and -r

`-r` is well known, so I'll leave it as-is, I expanded `-t` to `--target`.